CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Updated Browser Compatibility section in `README.md` to better indicate IE11, Firefox support
 - Public: Update English copy text for error message shown when no document is in the cameras view
 - Public: The `useMultipleSelfieCapture` configuration option is now stable and enabled by default
+- Public: Moved `UserAnalytics` event firing outside of `disableAnalytics` config check
```

Comment:
This should be moved under the next-version header

Reply:
Updated